build.cake

```diff
@@ -213,7 +213,7 @@ Task("IntegrationTest")
 			NUNIT3_CONSOLE,
 			new ProcessSettings()
 			{
-				Arguments = INTEGRATION_TEST_ASSEMBLY
+				Arguments = INTEGRATION_TEST_ASSEMBLY + " --where cat==teamcity"
```

Comment:
Yes we have a lot of tests for base scenarios. But also we have another points of integration which are not related to TeamCity service messages, for example the usage of "--where", specific exit codes of errors, getting of the list of tests, running console from the different locations, usage cmd arguments "process", "domain", "agents", "platform" and so on